aws/data_source_aws_instance_test.go

```diff
@@ -216,6 +216,84 @@ func TestAccAWSInstanceDataSource_VPCSecurityGroups(t *testing.T) {
 	})
 }
 
+func TestAccAWSInstanceDataSource_getPasswordData_true(t *testing.T) {
```

Comment:
Can we reduce these down to two acceptance tests please? true -> false and false -> true should be sufficient